lua/astrocommunity/utility/nvim-toggler/init.lua

```diff
@@ -0,0 +1 @@
+return { "nguyenvukhang/nvim-toggler", event = "User Astrofile" }
```

Comment:
Also maybe `event = { "User AstroFile", "InsertEnter" }` 

At least a fix for `Astrofile` -> `AstroFile`

Reply:
Fixed